tests/auth-admin.test.js

```diff
@@ -19,9 +19,15 @@ describe('Auth for admin users', () => {
   describe('Query.users', () => {
     beforeAll(async () => {
       const Users = db.collection('users');
+
+      Users.createIndex({
```

Comment:
Tests should not create indexes, I checked and I've seen that there is also a createIndex in the products seeds but I don't get why, maybe the important thing is awaiting it?

Reply:
Because there're tests that utilize the search functionality and they don't seem to work unless we set the indexes.